src/sql/azdata.d.ts

```diff
@@ -3057,6 +3058,12 @@ declare module 'azdata' {
 		CSSStyles?: { [key: string]: string };
 	}
 
+	export interface ImageComponentProperties {
+		src: string;
```

Comment:
{light: string; dark:string} | string?

Reply:
Currently this doesn't support different light/dark images since I didn't need it for what I was doing. Given that no one is going to be using this I won't add that functionality yet - if we end up needing it in the future we can add it then.